code/src/WijmoProvider/Features/ColumnFilter.ts

```diff
@@ -80,7 +80,14 @@ namespace WijmoProvider.Feature {
         }
 
         public activate(columnID: string): void {
-            this.changeFilterType(columnID, this.filterType);
+            const column = this._grid.getColumn(columnID);
+            if (column) {
+                this.changeFilterType(columnID, this.filterType);
+            } else {
+                throw new Error(
+                    OSFramework.Enum.ErrorMessages.InvalidColumnIdentifier
+                );
+            }
         }
 
         public build(): void {
```

Comment:
@joselrio This method should be documented

Reply:
Done!